src/apps/copilots/src/pages/copilot-opportunity-list/index.tsx

```diff
@@ -88,6 +90,9 @@ const tableColumns: TableColumn<CopilotOpportunity>[] = [
 const CopilotOpportunityList: FC<{}> = () => {
     const navigate = useNavigate()
 
+    const { profile }: ProfileContextData = useContext(profileContext)
+    const isAdmin: boolean = useMemo(() => !!profile?.roles?.includes(UserRole.tcaAdmin), [profile])
```

Comment:
@himaniraghav3 can we please extend this to PM role to ensure PMs can use the functionality as well?